### ISSUE TITLE
Clarity between report and export

### ISSUE THREAD

**Issue by hrj** [hrj/abandon#71]
Following the discussion from PR #70, I think abandon should be more consistent in how configurations are named. I have been loosely following this idea:

There are two ways to get data out of abandon: `report` and `export`.
* `report` will show processed data from abandon with some visual formatting.
* `export` will emit raw data in some standard format (currently `xml` and `ledger`) 

However, that is probably not a good way to define `report` and `export`. Sometimes you need processed data in a standard format, or raw data shown with some visual formatting.

Hence, it would be better to separate the "processed" / "raw" distinction from the "report" / "export" distinction. **Here's the new plan:**

* `report` is for any visually formatted output, intended for direct human consumption. 
* `export` is for non-visual output, intended to be machine readable.

Reports are in textual format, for now. In addition, the GUI can show the reports in an interactive format.

The current export configurations would need to be expressed differently:

| Current config | Description | New config|
|----|----|----|
|`exports += {type = ledger}`| Exports the balance of accounts in ledger format| `exports += {type = balance, format = ledger}`|
|`exports += {type = xml}`| Exports transactions in xml format| `exports += {type = transactions, format = xml}`|

With this change, two new types of exports that will also be supported:
`exports += {type = balance, format = xml}`
and
`exports += {type = transactions, format = ledger}`

----

To summarize: the distinction between report and export will be clearer; report is for human consumption while export is for machine-readable output. Both reports and exports will have two parameters: a `type` and a `format`. The format for `report` is currently implied to be text, while for `export` it will be specified in the configuration.

**Comment by hrj** [comment]
I believe this has been addressed in https://github.com/hrj/abandon/pull/70

Closing.
